packages/design-system/src/styles/components/_LabelMask.scss

```diff
@@ -1,4 +1,4 @@
 .ds-c-label-mask {
   color: var(--color-muted);
-  font-family: Menlo, Consolas, Monaco, Liberation Mono, Lucida Console, monospace;
+  font-family: Menlo, Consolas, Monaco, 'Liberation Mono', 'Lucida Console', monospace;
```

Comment:
Can't believe these strings weren't caught before - multi-word font families won't work right unless wrapped in quotes

Reply:
😮